library/src/pivotal-ui/components/dropdowns/index.js

```diff
@@ -1,5 +1,4 @@
 try {
-  require('pui-css-bootstrap');
```

Comment:
I believe we are still using the bootstrap css, we just aren't using React Bootstrap